k8s/common/omsagent-configmap.yaml

```diff
@@ -1,124 +0,0 @@
-kind: ConfigMap
```

Comment:
probably shouldn't be in common but again shouldn't be deleted as part of this

Reply:
Agreed.